templates/aws/pro.html

```diff
@@ -0,0 +1,267 @@
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Priced for the cloud</h3>
+          <div class="p-matrix__desc">
+            <p>Ubuntu Pro pricing tracks the underlying EC2 cost, starting at 20% for the smallest t3.nano instances, and ramping down to less than 1%.</p>
+          </div>
+        </div>
+      </li>
+    </ul>
+    <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+    <p><a href="/security">Learn more about security for Ubuntu&nbsp;&rsaquo;</a></p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <h2>Launch Ubuntu Pro for AWS</h2>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 18.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 16.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 14.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821V7QJP" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Broadest open source security coverage keeps CISOs sleeping at night</h2>
+      <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. Ideal for companies that have embraced open source for their new products.</p>
+      <p>Developers choose Ubuntu because it includes a set of open source packages known as ‘universe’. These include NGINX, MongoDB, Redis, Kafka&hellip; .With Ubuntu Pro, CIOs and security officers can be confident they can roll these packages out to production.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```

Reply:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.